support-frontend/assets/helpers/__tests__/theGridTest.ts

```diff
@@ -1,23 +1,18 @@
 // ----- Imports ----- //
-import {
-	GRID_DOMAIN,
-	gridSrcset,
-	gridUrl,
-	imageCatalogue,
-} from '../images/theGrid';
+import { GRID_DOMAIN, gridSrcset, gridUrl } from '../images/theGrid';
 // ----- Tests ----- //
 describe('theGrid', () => {
 	it('should construct a grid url', () => {
-		const gridId = 'guardianObserverOffice';
+		const gridId = 'weeklyLandingHero';
 		const size = 333;
-		const expectedUrl = `${GRID_DOMAIN}/${imageCatalogue[gridId]}/333.jpg`;
+		const expectedUrl = `${GRID_DOMAIN}/img/media/0e5187e523a6445003fae7b1bb4124a8a4a814c8/0_655_6362_2706/master/333.jpg?dpr=1&s=none&width=333`;
```

Comment:
Is there a need to hardcode the ID here, or could we continue to use ``${imageCatalogue[gridId]``?

Reply:
Just habit of writing tests to not couple them to any implementation. For example the last test was actually returning `undefined`, which, while valid, felt odd.

Happy to change it back.

Reply:
I'd consider this still tied to the implementation in that if we ever change or remove the `weeklyLandingHero` id the test will break even if there's nothing wrong with the code, but it's up to you.

Reply:
I'll leave it as is as it seems both ways seem to make sense.

My logic is "Given these configuration options, `size` and `id`, I expect this answer, whereas if you lookup the URL in the expected and test cases, you're essentially ignoring it as a parameter to `theGrid`.

I think I might be getting slightly overly-philosophical on the case, but interesting to think through.